src/main/frontend/handler/shell.cljs

```diff
@@ -33,22 +36,36 @@
      :success
      false)))
 
+(def dangerous-commands
```

Comment:
I'm concerned this list doesn't protect enough. Have we considered something like the shellquote npm lib - https://auth0.com/blog/preventing-command-injection-attacks-in-node-js-apps/#Preventing-Command-Injection?